packages/hdwallet-keepkey-webusb/src/adapter.ts

```diff
@@ -32,7 +32,7 @@ export const AdapterDelegate = {
       if (out.serialNumber === undefined) throw new Error("expected serial number");
       return out as Device;
     } catch (e) {
-      throw new core.WebUSBCouldNotPair("KeepKey", e.message);
+      throw new core.WebUSBCouldNotPair("KeepKey", String(core.isIndexable(e) ? e.message : e));
```

Comment:
nit: I'd prefer you just import the function and use it as `isIndexable` instead of `core.isIndexable`.

Reply:
We should move the function out of `hdwallet-core` to a utility package at some point, but I od feel `import { * as core, isIndexable } from "@shapeshiftoss/hdwallet-core";` would be the more awkward syntax.